cloudify/utils.py

```diff
@@ -223,6 +223,15 @@ def get_manager_file_server_root():
     return os.environ[constants.MANAGER_FILE_SERVER_ROOT_KEY]
 
 
+def get_manager_rest_service_protocol():
+    protocol = os.environ[constants.REST_PROTOCOL_KEY]
+
+    if protocol is None:
+        protocol = constants.SECURED_PROTOCOL
+
+    return protocol
```

Comment:
I note that `os.environ[nonexistent]` throws a KeyError, so the `is None` check is futile.
This perhaps just wants to be:
```suggestion
    return os.environ.get(
        constants.REST_PROTOCOL_KEY,
        constants.SECURED_PROTOCOL,
    )
```

(same for the other func)